csharp/Platform.Data.Doublets/ILinksExtensions.cs

```diff
@@ -878,14 +878,20 @@ public static TLink SearchOrDefault<TLink>(this ILinks<TLink> links, TLink sourc
 
         /// <param name="links">Хранилище связей.</param>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static TLink Create<TLink>(this ILinks<TLink> links) => links.Create(null);
+        public static TLink Create<TLink>(this ILinks<TLink> links) => links.Create(null, null);
+
+        public static TLink Create<TLink>(this ILinks<TLink> links, IList<TLink> substitution) => links.Create(substitution, null);
+
+        public static TLink CreatePoint<TLink>(this ILinks<TLink> links) => CreatePoint(links, null);
```

Comment:
You should define your own handler here and return the address of the created link here. Otherwise, the code that expected to get created link address from this method will break.